### PR TITLE
Check weather given string is palindrome or not

### PR DESCRIPTION
**Pull request by Aarti002**
issue #346 solved
Python
Doubly linked list

### CODE REVIEW

**Comment by ANAMIKA1410** [comment]
hello ;
I'm Anamika jain , GSSoC participant
please assign this to me @Aarti002 @anubhavitis @aaadddiii @tarun26091999 @dheerajkotwani

**Review by ANAMIKA1410 — COMMENTED** [review]
def isPalindrome(str):
 
    # Run loop from 0 to len/2 
    for i in range(0, int(len(str)/2)): 
        if str[i] != str[len(str)-i-1]:
            return False
    return True

**Comment by Aarti002** [comment]
hey @ANAMIKA1410 i'm a participant as well :)

**Comment by krishantraj** [comment]
can I solve this issue #1239 in java without using a linked list? if yes so please assign it to me.

File: Linked_list/Python/isPalindrom_dll.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+
+if __name__=="__main__":
+    s=input('Enter string: ')
+
+    items = doubly_linked_list()
+    for i in s:
+        items.append_item(i)
+    print(items.isPalindrom())
+    #items.print_foward()
```
Comment (plazzy99) [inline Linked_list/Python/isPalindrom_dll.py:68]: time and space complexity required at bottom

**Review by vatsalkesarwani12 — CHANGES_REQUESTED** [review]
update readme

**Comment by Aarti002** [comment]
@plazzy99 i have a doubt where we should keep this solutions file? in "Code"  folder or topic related folder like in this case "Linked-List", please help!
Thank you!

**Comment by plazzy99** [comment]
> @plazzy99 i have a doubt where we should keep this solutions file? in "Code" folder or topic related folder like in this case "Linked-List", please help!
> Thank you!

https://github.com/Algo-Phantoms/Algo-Tree/issues/1202

**Comment by Aarti002** [comment]
@plazzy99 please check once i have added my code link in readme file
Thank you!

**Comment by rudrakshi99** [comment]
Please resolve merge conflicts.

**Comment by tarun26091999** [comment]
@rudrakshi99 look into it.

**Pull request closed without merging**